core/src/main/java/edu/wpi/grip/core/Palette.java

```diff
@@ -31,7 +32,21 @@
     @Subscribe
     public void onOperationAdded(OperationAddedEvent event) {
         final Operation operation = event.getOperation();
-        this.operations.put(operation.getName(), operation);
+        map(operation.getName(), operation);
+        for(String alias : operation.getAliases()) {
+            map(alias, operation);
+        }
+    }
+
+    /**
+     * Maps the key to the given operation
+     * @param key The key the operation should be mapped to
+     * @param operation The operation to map the key to
+     * @throws IllegalArgumentException if the key is already in the {@link #operations} map.
+     */
+    private void map(String key, Operation operation) {
```

Comment:
> Why and how? Put the same code twice?

Yes.  It's literally just two lines of code.  You're actually using *more* lines to separate it into a separate method than it would take to just call `operations.put` twice.